Source/santabs/SNTBundleService.m

```diff
@@ -25,6 +25,22 @@
 #import "SNTXPCConnection.h"
 #import "SNTXPCNotifierInterface.h"
 
+// For the SNTBundleService class, make SNTStoredEvent objects equal if the SHA256 matches.
+@implementation SNTStoredEvent (BundleEquatable)
```

Comment:
Good catch on this, makes sense.

The subclass approach would require larger refactoring. XPC drops the replies since the collection member type doesn't match anymore.

I would like to keep this compare behavior just to santabs. The dictionary approach seems like a reasonable way forward.